wallet/txauthor/go.mod

```diff
@@ -4,7 +4,18 @@ go 1.12
 
 require (
 	github.com/btcsuite/btcd v0.22.0-beta.0.20220204213055-eaf0459ff879
+	github.com/btcsuite/btcd/btcec/v2 v2.1.0
 	github.com/btcsuite/btcd/btcutil v1.1.0
 	github.com/btcsuite/btcwallet/wallet/txrules v1.2.0
 	github.com/btcsuite/btcwallet/wallet/txsizes v1.1.0
 )
+
+replace github.com/btcsuite/btcd => github.com/guggero/btcd v0.20.1-beta.0.20220214155024-51fd177d32c0
+
+replace github.com/btcsuite/btcd/btcec/v2 => github.com/guggero/btcd/btcec/v2 v2.0.0-20220214155024-51fd177d32c0
+
+// The old version of ginko that's used in btcd imports an ancient version of
+// gopkg.in/fsnotify.v1 that isn't go mod compatible. We fix that import error
+// by replacing ginko (which is only a test library anyway) with a more recent
+// version.
+replace github.com/onsi/ginkgo => github.com/onsi/ginkgo v1.14.2
```

Comment:
Is this a permanent thing?

Reply:
Yeah, this is because of the ancient version of `goleveldb` that is used. Would be fixed by https://github.com/btcsuite/btcd/pull/1780 I think.